packages/openapi-ts/src/generate/schemas.ts

```diff
@@ -19,6 +19,20 @@ const ensureValidSchemaOutput = (
   }
 
   const result = { ...schema };
+
+  if ('oneOf' in result && Array.isArray(result.oneOf)) {
+    const oneOfArray = result.oneOf as any[];
+    const hasNull = oneOfArray.some((item) => item === null || item?.nullable);
+
+    if (hasNull) {
+      result.oneOf = oneOfArray.filter(
+        (item, index) =>
+          !(item === null || item?.nullable) ||
+          index === oneOfArray.findIndex((i) => i === null || i?.nullable),
+      );
+    }
+  }
```

Comment:
When I remove this code, the generated output doesn't change. Are you seeing something else on your end?